tutorials/cloud-iot-gateways-rpi/cloudiot_mqtt_gateway.py

```diff
@@ -32,13 +33,18 @@
 ADDR = (HOST, PORT)
 
 udpSerSock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
-udpSerSock.setblocking(0)
+udpSerSock.setblocking(False)
 udpSerSock.bind(ADDR)
 
+skip_next_sub = False
```

Comment:
Does this do anything? I can't seem to see where it's being used.

Reply:
This is unused, I'll fix :)